lgc/include/lgc/state/TargetInfo.h

```diff
@@ -105,7 +105,6 @@ struct WorkaroundFlags {
       unsigned disableI32ModToI16Mod : 1;
 
       unsigned waTessFactorBufferSizeLimitGeUtcl1Underflow : 1;
-      unsigned waTessIncorrectRelativeIndex : 1;
```

Comment:
Suggested by Qun before, we still keep the workaround flags here and just make them ineffective.

Reply:
OK!